ROADMAP.md

```diff
@@ -0,0 +1,30 @@
+![PODMAN logo](https://raw.githubusercontent.com/containers/common/main/logos/podman-logo-full-vert.png)
+
+# Podman Roadmap
+
+The Podman development team reviews feature requests from its various stakeholders for consideration
```

Comment:
I don't think we want to be this specific. We review priorities quarterly and make decisions on future project direction based on them; I don't think we need to be more specific than that.

Reply:
I would envision us linking GH or Jira requests that could have that information.